### ISSUE TITLE
Society Checking - 62475 - U.S. General

### ISSUE THREAD

**Issue by stellasilbert** [theglobaljukebox/cantometrics#96]
Three out of the four songs currently assigned to 62475, U.S. General, are also included in Jesse Rifkin's [Pop Song dataset](https://docs.google.com/spreadsheets/d/1BZn224xoL_0KF93KQPemDfeZqYXCruo9s2jjDx-SuCc/edit#gid=0) (an addition to the Urban Strain set). Since the society "U.S. General" seems quite broad and difficult to square with the rest of the Cantometrics dataset, and Jesse's more refined American subcultural groups seem more appropriate for classifying these songs, I suggest removing all four songs from the Cantometrics set, and deleting society 62475 U.S. General. The three songs that are already overlapping with the Pop set can stay there (Glory, Glory; Where Have All the Flowers Gone; Amazing Grace), and we can add the fourth (Joe Hill). All of these songs are currently uncoded, so as far as I can see nothing would be lost by removing them from Cantometrics.

<google-sheets-html-origin><style type="text/css"></style>

NC159c | Glory, Glory | Odetta Holmes, vocal
-- | -- | --
NC168c | Where Have All the Flowers Gone | Pete Seeger, vocal
NC54c | Amazing Grace | Judy Collins, lead vocal, choir
NC152c | Joe Hill | Paul Robeson, vocal, Alan Booth, piano

So, to summarize my suggestions:
1) Remove NC159c, NC168c, NC54c, and NC152c from Cantometrics dataset.
2) Remove society 62475, U.S. General, from society dataset.
3) Add song "Joe Hill" to Pop song dataset.

**Comment by stellasilbert** [comment]
All suggestions implemented
